### ISSUE TITLE
Item deletion glitch

### ISSUE THREAD

**Issue by Nike020410** [Wynncraft/Issues#8742]
The Accessory Slots dropped my rings and the bracelet from the morph set, which got deleted. Nowhere in the bank of Cinfras (where I was)  or near it I could find my items.
The inventory was filled.
(My Armor has higher requirements, but on intelligence, they did not disappear, once my items were dropped on login and strength is so far the only affected skill)
Screenshots: http://imgur.com/a/0ZCmu

**Comment by thecoolkogut4** [comment]
Normally the item goes above you. Even though you didn't ask I'd like to inform you that the server has a strict no-refund policy which means you won't be getting you items back.

**Comment by Nike020410** [comment]
"We" searched the second floor and I wanted to inform you, that items getting erased is not fun at all and should be fixed as soon as possible.

**Comment by thecoolkogut4** [comment]
After 5 minutes (I believe it's 5) all items on the ground will despawn. Is it possible that you waited more than 5min or that someone else picked it up?

**Comment by Nike020410** [comment]
I immediatly started searching, only a (very trustworthy) friend was there. I even looked up where they could have landed over my head and there was no second floor.

**Comment by thecoolkogut4** [comment]
There's also a possibility that it went through a wall, in the end you can't get it back. There's nothing you can do

**Comment by Nike020410** [comment]
TLDR: Screenshots should provide enough information and items will continue to be deleted and fml.

**Comment by thecoolkogut4** [comment]
All I can say, next time this happens look around everywhere and try to find it. Sorry.
